Examples/Genrefy/Genrefy/Activities/Onboarding/ImportingCoordinator.swift

```diff
@@ -34,6 +35,14 @@ class ImportingCoordinator: NSObject, ActivityCoordinating {
         self.repository = ImportRepository()
         super.init()
         
+        if
+            let tokenData = KeychainService.shared.loadEntry(for: "oAuthToken"),
+            let token = try? NSKeyedUnarchiver.unarchiveTopLevelObjectWithData(tokenData) as? DMEOAuthToken {
```

Comment:
You cn just use the normal `unarchiveObject(with: tokenData)` here.

Reply:
it will give deprecated message: 
`unarchiveObject(with:)' was deprecated in iOS 12.0: Use +unarchivedObjectOfClass:fromData:error: instead`